src/partials-js/chart.js

```diff
@@ -1,152 +1,274 @@
+      dew_point: -4.99,
+      wind_speed: 2.83,
+      wind_deg: 45,
+      wind_gust: 6.28,
       weather: [{ id: 803, main: 'Clouds', description: 'broken clouds', icon: '04d' }],
-      clouds: { all: 66 },
-      wind: { speed: 2.37, deg: 307, gust: 2.68 },
-      visibility: 10000,
+      clouds: 54,
       pop: 0,
-      sys: { pod: 'd' },
-      dt_txt: '2022-04-14 06:00:00',
+      uvi: 5,
+    },
+    {
+      dt: 1651226400,
+      sunrise: 1651201781,
+      sunset: 1651255007,
+      moonrise: 1651201080,
+      moonset: 1651249440,
+      moon_phase: 0.95,
+      temp: { day: 13.31, min: 4.74, max: 15.43, night: 11.36, eve: 14.93, morn: 4.74 },
+      feels_like: { day: 11.69, night: 10.12, eve: 13.58, morn: 2.97 },
+      pressure: 1030,
+      humidity: 38,
+      dew_point: -0.6,
+      wind_speed: 2.84,
+      wind_deg: 196,
+      wind_gust: 5.09,
+      weather: [{ id: 500, main: 'Rain', description: 'light rain', icon: '10d' }],
+      clouds: 18,
+      pop: 0.35,
+      rain: 0.11,
+      uvi: 5,
     },
     {
-      dt: 1649926800,
-      main: {
-        temp: 8.41,
-        feels_like: 8.41,
-        temp_min: 8.41,
-        temp_max: 8.41,
-        pressure: 1020,
-        sea_level: 1020,
-        grnd_level: 1020,
-        humidity: 74,
-        temp_kf: 0,
-      },
+      dt: 1651312800,
+      sunrise: 1651288063,
+      sunset: 1651341510,
+      moonrise: 1651288260,
+      moonset: 1651340400,
+      moon_phase: 0,
+      temp: { day: 15.91, min: 9.52, max: 16.05, night: 12.5, eve: 15.81, morn: 9.52 },
+      feels_like: { day: 14.65, night: 11.32, eve: 14.54, morn: 8.86 },
+      pressure: 1025,
+      humidity: 42,
+      dew_point: 3.14,
+      wind_speed: 2.6,
+      wind_deg: 222,
+      wind_gust: 4.14,
       weather: [{ id: 803, main: 'Clouds', description: 'broken clouds', icon: '04d' }],
-      clouds: { all: 73 },
-      wind: { speed: 0.37, deg: 110, gust: 0.77 },
-      visibility: 10000,
+      clouds: 74,
       pop: 0,
-      sys: { pod: 'd' },
-      dt_txt: '2022-04-14 09:00:00',
+      uvi: 5,
     },
   ],
-  city: {
-    id: 2641549,
-    name: 'Newtonhill',
-    coord: { lat: 57, lon: -2.15 },
-    country: 'GB',
-    population: 3284,
-    timezone: 3600,
-    sunrise: 1649826353,
-    sunset: 1649877119,
-  },
 };
-console.log(Object.values(testArray));
-//create chart
-const labels = ['January', 'February', 'March', 'April', 'May', 'June'];
-
+console.log(apiResponse);
+const labels = apiResponse.daily.map(element => {
+  return format(element.dt, 'PP');
+});
+const temp = apiResponse.daily.map(element => {
+  return element.temp.day;
+});
+const humidity = apiResponse.daily.map(element => {
+  return element.humidity;
+});
+const windSpeed = apiResponse.daily.map(element => {
+  return element.wind_speed;
+});
+const pressure = apiResponse.daily.map(element => {
+  return element.pressure;
+});
+const time = format(1650708000, 'LLLL d, R');
+console.log(time);
 const data = {
```

Comment:
jak dobrze rozumiem to jest struktura danych dla chart i wymaga ona aby np kolor tła był zdefiniowany tu w js ?

Reply:
To jest jeszcze nie skonczone bo nie miala jak powinno